### DISCUSSION TITLE
feature req add the engine version to title bar.

### DISCUSSION THREAD

**Discussion by steingrim** [godotengine/godot-proposals#5209] [Editor]
I'd like to see the version of Godot Engine in the title bars.

It would be helpful when following tutorials or when answering other people's questions to know which version they had used.

![image](https://user-images.githubusercontent.com/1104287/185645516-248689d3-33d2-4c4c-a180-edcbbe1973ea.png)

![image_2022-08-19_074358061](https://user-images.githubusercontent.com/1104287/185644325-bfab594f-716f-4419-9a4b-9f20498b4b7b.png)

**Comment by Calinou** [comment]
The engine version is already mentioned at the bottom of the editor and in the project manager.

I'd prefer not having the version number in the window title bar, as it'd make the already long title even longer.

**Reply by steingrim** [reply]
<img width="1278" alt="image" src="https://user-images.githubusercontent.com/1104287/185746616-26576891-9381-4413-b7df-89048e9b7578.png">

Am I blind?

**Reply by Zireael07** [reply]
"3.5 stable" next to the two arrows at the bottom right (below Copy/Clear)

**Reply by steingrim** [reply]
Thanks, I still find that some tutorials crop the screen.

**Reply by seppoday** [reply]
Some tutorials can also crop top of the screen :P

**Reply by Calinou** [reply]
> Thanks, I still find that some tutorials crop the screen.

I'd argue cropping the window title is going to be even more common compared to cropping the bottom of the window :slightly_smiling_face: 

The window title will always be cropped if you run the editor in fullscreen, or if you use any window capture tool while excluding the title bar.

**Reply by Zireael07** [reply]
@Calinou Misquoted here (you probably had two tabs open :P )

**Reply by steingrim** [reply]
Trying to follow a tutorial which is unfollowable in some cases doesn't seem a trivial thing.

This isn't going to impact the people working on the project or the people who tend to reply to bugs on github as much as it will a casual or first time user.

This change also doesn't require the version to be moved from the bottom, nor does it ask or suggest a move. It asks for information to be added to the tittle bar.

Another way to go might be to introduce a 'tutorial feature' as a deeper setting which adds useful non-production features. This could be a set of settings allowing the creator to select which additional information they'd like displayed and where, corner, splash screen, etc.

**Reply by timothyqiu** [reply]
> This could be a set of settings allowing the creator to select which additional information they'd like displayed and where, corner, splash screen, etc.

I think this would be more appropriate to add when editing the video, since it's just static text.
